### ISSUE TITLE
prepare our own dataset

### ISSUE THREAD

**Issue by zahrabsh74** [lorenzonava96/Landslide-mapping-on-SAR-data-by-Attention-U-Net#2]
well done for this helpful code. thank you so much for sharing it, but could you just explain how did you make your training dataset in 2 different X_train and Y_train parts? I just wanted to finetune this model on my own image from Sentinel1 data.

**Comment by lorenzonava96** [comment]
Hi Zahra,

This notebook will be super helpful to you:
https://github.com/bnsreenu/python_for_microscopists/blob/master/228_semantic_segmentation_of_aerial_imagery_using_unet/228_training_aerial_imagery.py

My advice is to clip both S1 and mask from QGIS (with the same spatial
resolution and number of pixels). then you can adapt this code.

Cheers
Lorenzo

Il giorno mer 12 lug 2023 alle ore 17:59 Zahra Badamchi <
***@***.***> ha scritto:

**Comment by zahrabsh74** [comment]
Dear Lorenzo, Thank you so much for your fast response and your nice help.
